apps/web/src/styles/features/home.css

```
@@ -123,22 +123,31 @@
+    );
   }
 
   .mobile-bottom-nav {
     position: fixed;
     right: 0;
-    bottom: 0;
+    /* Chrome's optimized edge-to-edge pattern: the surface grows once using
+       the maximum inset, then follows the dynamic chin using only `bottom`.
+       This avoids layout thrashing and removes the empty gesture-area band. */
+    bottom: calc(
+      env(safe-area-inset-bottom, 0px) -
```

Stylelint (17.14.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)